made-with.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default.html
+title: Made With Servo
+---
+
+<style type="text/css">
+
+.made-with-container {
+  gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 32px;
```

Comment:
margin-bottom not needed here (see other note about `subpage-content` class)

Reply:
It's still needed to separate sections. I've set it to 0 for `:last-child` so that the `subpage-content` margin is used there.